src/framework/audio/internal/platform/lin/linuxaudiodriver.cpp

```diff
@@ -88,8 +88,10 @@ static void alsaCleanup()
     if (_alsaData->threadHandle) {
         pthread_join(_alsaData->threadHandle, nullptr);
     }
-    snd_pcm_drain(_alsaData->alsaDeviceHandle);
-    snd_pcm_close(_alsaData->alsaDeviceHandle);
+    if (_alsaData->alsaDeviceHandle) {
```

Comment:
how is that related to tests w/o the "audio" module? Linux audio driver is a part of it

Reply:
I guess not. If so perhaps this is a quick-fix and the proper fix is to not load the linux-audio-module.
This patch addresses those users who either has a broken linux-audio stack, or purposefully run musescore without any loaded audiodriver.

Reply:
Further guessing the proper fix should take place in src/framework/audio/audiomodule.cpp, where we unconditionally load alsa if on linux. Until that is done, this patch is a stop-gap solution to avoid crashing.